content/kubermatic/main/installation/local-kkp-installation/_index.en.md

```diff
@@ -29,11 +29,26 @@ Follow these steps to use the KKP installer local command:
 
 **1. Download the installer.** This is the only manual step; there's no need to prepare any configuration since the installer should automatically configure KKP.
 
+{{< tabs name="Download the installer" >}}
+{{% tab name="Linux" %}}
 ```bash
 VERSION=$(curl -w '%{url_effective}' -I -L -s -S https://github.com/kubermatic/kubermatic/releases/latest -o /dev/null | sed -e 's|.*/v||')
 wget https://github.com/kubermatic/kubermatic/releases/download/v${VERSION}/kubermatic-ce-v${VERSION}-linux-amd64.tar.gz
 tar -xzvf kubermatic-ce-v${VERSION}-linux-amd64.tar.gz
 ```
+{{% /tab %}}
+{{% tab name="MacOS" %}}
+```bash
+# For latest version:
+VERSION=$(curl -w '%{url_effective}' -I -L -s -S https://github.com/kubermatic/kubermatic/releases/latest -o /dev/null | sed -e 's|.*/v||')
+# For specific version set it explicitly:
+# VERSION=2.21.x
+wget https://github.com/kubermatic/kubermatic/releases/download/v${VERSION}/kubermatic-ce-v${VERSION}-darwin-amd64.tar.gz
```

Comment:
nit: you probably want an `ARCH` variable defined before that gets the machine architecture, since macOS systems can either be AMD64 or ARM64.

Reply:
Good idea, I also thought about it when applying the same to the other installation [docs](https://github.com/kubermatic/docs/pull/1449)
Let me then extend this PR